arch/xtensa/src/esp32/esp32_wifi_adapter.c

```diff
@@ -638,26 +755,20 @@ static int esp_event_lock(bool lock)
 static void esp_set_isr(int32_t n, void *f, void *arg)
 {
   int ret;
+  uint32_t tmp;
   struct irq_adpt *adapter;
-  int irq;
-  int cpu = 0;
-  int tmp;
+  int irq = n + XTENSA_IRQ_FIRSTPERIPH;
 
-  if (g_wifi_irq >= 0)
-    {
-      return ;
-    }
+  wlinfo("INFO: n=%d f=%p arg=%p irq=%d\n", n, f, arg, irq);
 
-  irq = esp32_alloc_levelint(1);
```

Comment:
Where is the `alloc_levelint` now?

Reply:
The wifi will directly register peripheral to CPU interrupt in `wifi_set_intr`.

Reply:
The wifi will directly register peripheral to CPU interrupt in `wifi_set_intr`.

Reply:
Ok.